modules/stanford_layout_paragraphs/src/EventSubscriber/StanfordLayoutParagraphsSubscriber.php

```diff
@@ -33,7 +33,9 @@ public function layoutParagraphsAllowedTypes(LayoutParagraphsAllowedTypesEvent $
     if ($parent_component) {
 
       $layout_settings = $parent_component->getSettings();
-      if ($layout_settings['layout'] != 'layout_paragraphs_1_column') {
+      $layout_regions = \Drupal::service('plugin.manager.core.layout')
```

Comment:
This is a good plan. But the service should be loaded via dependency injection, using the services.yml file and a constructor method in this class.